dashboard/app/models/pd/teacher_application.rb

```diff
@@ -95,9 +95,15 @@ def validate_required_application_fields
     end
   end
 
+  before_validation :downcase_emails
+  def downcase_emails
```

Comment:
mehal+teacher@gmail.com and mehal+employee@gmail.com are the same email address as well. Should we apply filtering for that as well?